src/EditorFeatures/CSharpTest/PullMemberUp/CSharpPullMemberUpTests.cs

```diff
@@ -0,0 +1,681 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
Do we have "pull up members" 'cross language tests'?  We need to validate that we at least do not *crash* in cross language scenarios (i.e. C# pulling up into VB, and VB pulling up into C#).  It would also be nice to make sure this works properly, but that's less important than just not crashing :)

Reply:
:)